2024/Metadata-Codesprint-Nov24/engineering-report/sections/results/geonetwork.adoc

```diff
@@ -0,0 +1,25 @@
+[[results_Geonetwork]]
+==== OSGeo GeoNetwork
+
+===== GeoNetwork 5 DCAT-AP and OGCAPI-Records
+
+The GeoNetwork project recently started development of https://github.com/geonetwork/geonetwork[GeoNetwork 5].  This is a new GeoNetwork architecture based on the current, industry-leading, https://github.com/geonetwork/core-geonetwork/[GeoNetwork 4].  It incorporates ideas experimented in the https://github.com/geonetwork/geonetwork-microservices[GeoNetwork Microservices] project.
+
+The two main goals of this sprint was to start development on an OGCAPI-Records implementation and country specific DCAT-AP and GeoDCAT outputs.  Since the GeoNetwork 5 project is so new, these functionalities will be first-class in the architecture.  
+
+Several GeoNetwork core committers are supporting this work - Jeroen Ticheler (GeoCat), Francois Prunayre (CampToCamp), Jose Garcia (GeoCat), and David Blasby (GeoCat).  This code sprint was very beneficial because it gave an opportunity for these developers to communicate with other stakeholders and learn of other opportunities, similar work, and common issues.  This has already greatly improve the architecture, direction, and implementation.
```

Comment:
```suggestion
Several GeoNetwork core committers are supporting this work - Jeroen Ticheler (GeoCat), Francois Prunayre (titellus), Jose Garcia (GeoCat), and David Blasby (GeoCat).  This code sprint was very beneficial because it gave an opportunity for these developers to communicate with other stakeholders and learn of other opportunities, similar work, and common issues.  This has already greatly improve the architecture, direction, and implementation.
```